evcxr/src/command_context.rs

```diff
@@ -185,13 +185,17 @@ Panic detected. Here's some useful information if you're filing a bug report.
 
                 let mut eval_outputs = EvalOutputs::default();
 
-                // Add the stdout and stderr to eval_outputs
-                eval_outputs
-                    .content_by_mime_type
-                    .insert("stdout".to_string(), stdout_str.into());
+                let combined_output = if stderr_str.is_empty() {
+                    stdout_str.into_owned()
+                } else if stdout_str.is_empty() {
+                    stderr_str.clone().into_owned()
+                } else {
+                    format!("{}{}", stdout_str, stderr_str)
```

Comment:
It's probably simplest to just always do this - i.e. remove the above if and if-else.